index.html

```diff
@@ -14,46 +14,58 @@
 <body>
     <header id="header">
         <a href="#" class="logo">My Logo</a>
-        <nav>
+        <nav class="mobile-nav">
             <img src="./images/Enabled.png" alt="humburger menu">
         </nav>
+        <nav class="desk-nav">
+            <ul>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+
         
     </header>
 
 <section id="work_section">
     <section id="home_section">
         <div class="home_container">
-        <h1>I’m Shahnawaz Ali
-            Glad to see you!</h1>
-    
-            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <h1>I’m Shahnawaz Ali <br>
+                Glad to see you!</h1>
+        
+            <p class="frist_paraa">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
             <span>LET’S CONNECT</span>
-    
-                <ul class="social_media">
-                    <li><a href="https://twitter.com/RjShahnawaza75"><img src="./images/twitter.svg" alt="twitter"></a></li>
-                    <li><a href="https://www.linkedin.com/in/shahnawaz-ali-a24b72204" ><img src="./images/Linkedin_icon.svg" alt="linkedin"></a></li>
-                    <li><a href="#"><img src="./images/images4.svg" alt="ok"></a></li>
-                    <li><a href="#"><img src="./images/images5.svg" alt="github"></a></li>
-                    <li><a href="#"><img src="./images/images6.svg" alt="medium"></a></li>
-    
-                </ul>
-            </div>
+        
+            <ul class="social_media">
+                <li><a href="https://twitter.com/RjShahnawaza75"><img src="./images/twitter.svg" alt="twitter"></a></li>
+                <li><a href="https://www.linkedin.com/in/shahnawaz-ali-a24b72204" ><img src="./images/Linkedin_icon.svg" alt="linkedin"></a></li>
+                <li><a href="#"><img src="./images/images4.svg" alt="ok"></a></li>
+                <li><a href="#"><img src="./images/images5.svg" alt="github"></a></li>
+                <li><a href="#"><img src="./images/images6.svg" alt="medium"></a></li>
+        
+            </ul>
+        </div>
     </section>
-    
+    <div class="card_container4">
```

Comment:
## OPTIONAL

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_23](https://user-images.githubusercontent.com/69638013/160253658-71006d64-8485-4518-b577-f380b77b51f6.png) | ![Screenshot_22](https://user-images.githubusercontent.com/69638013/160253656-fdd39029-fd1d-4dac-9fda-70b7b7923dd1.png)

- [ ] Kindly justify the text in the project cards to look like the figma design, check screenshot above ⬆️ 
- [ ] Kindly switch the second card's image  to be on the right and text to be on the left so it  looks like the 4th project card